### PR TITLE
Update even or odd README.md

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/python/python-playground-questions/arithmetic-operation/arithmetic-operation.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-playground-questions/arithmetic-operation/arithmetic-operation.md?ref=7812b6bb1c21f42987c0112d257c0b89701288bd)

File: python/python-core/arithmetic-operation-exercise/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-name: Arithmetic Operations
+name: Multiplication & Division
 
-description: Create a program to either add, subtract, multiply or devide, two numbers.
+description: Create a program to either multiply or devide, two numbers.
```
Comment (kapnobatai137) [inline python/python-core/arithmetic-operation-exercise/README.md:3]: small typo
```suggestion
description: Create a program to either multiply or divide two numbers.
```

**Review by kapnobatai137 — CHANGES_REQUESTED** [review]

**Review by kapnobatai137 — APPROVED** [review]
:rocket:

**Pull request merged**